Archive synapse samples, update readme to remove "Modern Data Warehouse" terminology, replace with "Medallion" (#757)
Azure-Samples/modern-data-warehouse-dataops: README.md

- azure-databricks
- azure-stream-analytics
- azure-synapse-analytics
description: "Code samples showcasing how to apply DevOps concepts to common data engineering patterns and architectures leveraging different Microsoft data platform technologies."
---

# DataOps

> This repository contains numerous code samples and artifacts on how to apply DevOps principles to common data engineering patterns and architectures utilizing Microsoft Data Platform technologies.

Author: Done

> The samples are either focused on a single azure service (**Single Tech Samples**) or showcases an end to end data pipeline solution as a reference implementation (**End to End Samples**). Each sample contains code and artifacts relating one or more of the following capabilities:

Reviewer: ```suggestion
The samples are either focused on a single microsoft service (**Single-Technology Samples**) or showcases an end-to-end data pipeline solution as a reference implementation (**End-to-End Samples**). Each sample contains code and artifacts related to one or more of the following capabilities:
```

> ## Single Technology Samples

Reviewer: ```suggestion
## Single-Technology Samples
```

| Technology | Samples |
| ---------- | ------- |
| [Microsoft Fabric](./single_tech_samples/fabric/README.md) | ▪️ [CI/CD for Microsoft Fabric](./single_tech_samples/fabric/fabric_ci_cd/README.md)<br> ▪️ [Feature engineering on Microsoft Fabric](./single_tech_samples/fabric/feature_engineering_on_fabric/README.md) |
| [Azure SQL database](./single_tech_samples/azuresql/README.md) | ▪️ [CI/CD for Azure SQL database](./single_tech_samples/azuresql/azuresql_ci_cd/README.md) |
| [Azure Databricks](single_tech_samples/databricks/) | ▪️ [CI/CD for Azure Databricks](single_tech_samples/databricks/databricks_ci_cd/README.md) |
| [Azure Data Factory](./single_tech_samples/datafactory/README.md) | ▪️ [CI/CD for ADF with Auto publish](./single_tech_samples/datafactory/adf_cicd_auto_publish/README.md)<br> ▪️ [Data pre-processing using Azure Batch](./single_tech_samples/datafactory/adf_data_pre_processing_with_azure_batch/README.md) |
| [Azure Stream Analytics](./single_tech_samples/streamanalytics/README.md) | ▪️ [CI/CD for Azure Stream Analytics](./single_tech_samples/streamanalytics/streamanalytics_ci_cd/README.md) |

> ## End to End samples

Reviewer: ```suggestion
## End-to-End Samples
```

### DataOps for Medallion with Azure Data Factory and Azure Databricks

> This [sample](e2e_samples/parking_sensors/) demonstrates batch, end-to-end data pipeline utilizing Azure Data Factory and Azure Databricks built according to the Medallion architecture, along with a corresponding CI/CD process, Observability and automated testing.

Reviewer: We can explain in 2-3 sentences what Medallion architecture is.
or
At least, provide hyperlink to the doco:
[Medallion architecture](https://learn.microsoft.com/en-us/azure/databricks/lakehouse/medallion)

Author: Added

> This [sample](e2e_samples/parking_sensors/) demonstrates batch, end-to-end data pipeline utilizing Azure Data Factory and Azure Databricks built according to the [Medallion architecture](https://learn.microsoft.com/en-us/azure/databricks/lakehouse/medallion), along with a corresponding CI/CD process, Observability and automated testing.

Reviewer: ```suggestion
This [sample](e2e_samples/parking_sensors/) demonstrates batch, end-to-end data pipeline utilizing Azure Data Factory and Azure Databricks built according to the [medallion architecture](https://learn.microsoft.com/en-us/azure/databricks/lakehouse/medallion), along with a corresponding CI/CD process, observability and automated testing.
```
